### PR TITLE
docs: add week 5 todos for leads

### PR DESCRIPTION
**Pull request by tancnle**
# Description

Add Week 5 Todos for Leads

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## The story so far
+
+By now your entire team should have done a team retrospective. Meetings are hard
+so please try to think of ways to collaborate async with your team members. For
+example, use Miro to collect ideas async and shorten meetings with clear agenda.
+
+Please continue to promote your members efforts using the 'Shoutout!' channel or
+team meetings. It helps to drive engagement and improve team morale.
+
+## Planing
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: Small spelling error

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Please organise time for presentation and preparing the materials for the
+Showcase this week.
+
+Remind team to start looking at the Software Requirements Specification in
+preparation for OnTrack Task 6.2.
+
+Continue to build out your Trello boards. For teams who are still struggling
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md:24]: We will also be starting to provide some specific feedback on your Trello boards this week

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+If you are would like to improve sprint planning, the upcoming delivery lead
+workshop might be of great interest. We have rescheduled it from 6 - 7PM on
+Tuesday 05/04/2022.
+
+Guidance on user stories and requirement extracting will also be coming later
+this week.
+
+## Calendar
+
+Please use our company shared calendar
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: Please ensure your team's regular meetings are in the Thoth Tech calendar. This provides visibility and also means that leadership know when your meetings are incase we need to jump in to support

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+## Leadership support (you):
+
+You could help the company to help us understand the company learning and
+development needs and plan for resources. Please advocate the
+[survey](https://forms.office.com/Pages/ResponsePage.aspx?id=7Hgj0IgW1UaFQBwotfRw9pxmFQ5mP6pJsCNkD-HdouBUNUVBVlBEMlVTT0RKWkdVNTJOT0o5MTNMTSQlQCN0PWcu)
+to your team. If any team members would like to facilitate any training or you
+have identified any experts that can spread knowledge beyond your teams, please
+do not hesitate to reach out to the Area Leads. We love to collaborate and build
+a great learning culture at Thoth-tech.
+
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md:55]: If you would like to express interest to attend the leadership conference - please let us know. This is a shadowing development experience that would be beneficial, particularly for juniors taking more leadership responsibility next trimester

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+so please try to think of ways to collaborate async with your team members. For
+example, use Miro to collect ideas async and shorten meetings with clear agenda.
+
+Please continue to promote your members efforts using the 'Shoutout!' channel or
+team meetings. It helps to drive engagement and improve team morale.
+
+## Planing
+
+Please organise time for presentation and preparing the materials for the
+Showcase this week.
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: note - this is not compulsory - just an opportunity for students aiming for higher marks to showcase your team's work to the managing directors. You need to ensure your team are aware of this, and communicate to us who is interested and what time so we can schedule it it, however you do not need to participate if you do not want to.

Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md (hunk 4 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md:55]: Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md (hunk 3 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md (hunk 2 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md:24]: Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md (hunk 1 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md]: Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)

**Comment by tancnle** [comment]
Thank you for reviewing @ginge270. I have incorporated your feedback into this PR. Please let me know what you think 🙏🏼

**Review by ginge270 — APPROVED** [review]

**Pull request merged**